### PR TITLE
Prep

### PR DESCRIPTION
**Pull request by jburel**
Prepare the repository ahead of the first release and publication to pypi

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 from setuptools import setup, find_packages
 
+VERSION = '0.1.0.dev0'
```
Comment (will-moore) [inline setup.py:3]: does bumpversion update this, and the version in .bumpversion.cfg?

Comment (jburel) [reply setup.py:3]: it should see https://github.com/ome/omero-virtual-microscope for example

File: LICENSE [hunk 2]
```diff
@@ -1,23 +1,21 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
+NU AFFERO GENERAL PUBLIC LICENSE
```
Comment (pwalczysko) [inline LICENSE:1]:
```suggestion
GNU AFFERO GENERAL PUBLIC LICENSE
```

Maybe the G got lost during copy/paste ?

Comment (jburel) [reply LICENSE:1]: thanks. A letter got dropped during copy/paste action

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Configuration settings:
+
+::
+
+    $ omero config append omero.web.apps '"omero_web_zarr"'
+
+    # Allow to open-with Vizarr
+
+    $ omero config append omero.web.open_with '["web_zarr_vizarr", "omero_web_zarr_index", {"supported_objects":["image"], "label": "Vizarr", "script_url": "omero_web_zarr/openwith.js"}]'
+
```
Comment (pwalczysko) [inline README.rst:42]: Although open with is there in my locally run web, and it opens a new tab, it was black. @will-moore is telling me open with is broken atm

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+::
+
+    $ omero web restart
+
+
+This appli self-hosts Vizarr to avoid CORS issues (delegating to https://hms-dbmi.github.io/vizarr/).
+
+In the webclient UI you can use the context menu to `Open With > Vizarr`, or use your Image ID and go directly to:
+
```
Comment (pwalczysko) [inline README.rst:53]: ::

**Comment by pwalczysko** [comment]
Checked mainly the README and the licence file (what I understood). Gave it a short functional test as indicated https://github.com/ome/omero-web-zarr/pull/4#discussion_r823843287 and https://github.com/ome/omero-web-zarr/pull/4#discussion_r823844112 which seems to have gone well.

Found several minor issues in README

**Comment by will-moore** [comment]
I'm not sure how to fix the open-with issue at the moment. The JavaScript that generates the link doesn't have access to the absolute URL that vizarr now needs.
So maybe best to simply remove the `openwith` config and instructions from the README for now, until I can work out a fix.
I think `appli` -> `app` is the simplest typo fix.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-plugins-push#1424](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-plugins-push/1424/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-plugins-push/1424/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - README.md

--conflicts

**Pull request closed without merging**